spec/system/avo/trix_field_spec.rb

```diff
@@ -35,6 +35,13 @@
 
         expect(find_field_value_element("body")).to have_text "Works for us!!!"
       end
+
+      it "contains js alert messages translated" do
```

Comment:
@Paul-Bob it seems the data attribute it not showing in the test...do you know why?

Reply:
I'm not sure why, @rafaeliga. Have you checked the DOM locally? Is it rendered on DOM, like you're trying to check it on the test?

Reply:
Oops, you are right, I was checking the wrong data key. Thanks!